idaes/generic_models/properties/helmholtz/helmholtz.py

```diff
@@ -563,7 +563,9 @@ class HelmholtzParameterBlockData(PhysicalParameterBlock):
             default=None,
             domain=tuple,
             description="Tuple providing default temperature bounds",
-            doc="Tuple providing default temperature bounds",
+            doc="""This is an optional tuple providing default temperature bounds.
+The elements of the tuple should include units of temperature.
```

Comment:
It might just be the speed at which I skimmed the docs, but I didn't see an explicit example of what this tuple should look like i.e. is it `(lower, upper, units)` or something else?

Reply:
Okay, should be all fixed up.  You were right about it being unclear.  I make the documentation of the tuple format explicit and included an example.